test-data/unit/check-flags.test

```diff
@@ -898,3 +898,27 @@ Movie = TypedDict('Movie', {'name': str, 'year': int})
 def g(m: Movie) -> Movie:
     return m
 [builtins fixtures/dict.pyi]
+
+[case testDisallowPartiallyAnnotatedDefs]
```

Comment:
This is missing a test case for a function with one annotated argument, one unannotated argument, and no return type annotation.